### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by luke-at-mosaica** [sanity-io/sanity#7574] [Sanity Studio Feedback]
Hi Team,

Would love to see the Sanity Studio Portable text editor support toolbar shortcut keys on annotation components - our content team would LOVE to speed up their workflows with this.

Many thanks

**Comment by mornir** [comment]
I think this is now possible with the latest version (v3.92.0) of Sanity:
Tutorial: https://www.sanity.io/docs/studio/pte-plugins
Example plugin with keyboard shortcuts: https://github.com/portabletext/plugins/tree/main/plugins/markdown-shortcuts
